plugins/services/src/js/pages/ServicesPage.js

```diff
@@ -37,7 +37,7 @@ var ServicesPage = createReactClass({
 
   componentWillMount() {
     this.store_listeners = [
-      { name: "notification", events: ["change"], suppressUpdate: false }
```

Comment:
No idea which errors [this commit](https://github.com/dcos/dcos-ui/commit/b73c4540f9133b47af0b3ecacab73b3180415452) talks about, I found no data requirement